src/plot_api/subroutines.js

```diff
@@ -438,14 +514,7 @@ function getMainTitleX(fullLayout, textAnchor) {
 function getMainTitleY(fullLayout, dy) {
     var title = fullLayout.title;
     var gs = fullLayout._size;
-    var vPadShift = 0;
-
-    if(dy === '0em' || !dy) {
-        vPadShift = -title.pad.b;
-    } else if(dy === alignmentConstants.CAP_SHIFT + 'em') {
-        vPadShift = title.pad.t;
-    }
-
+    var vPadShift = getVPadShift(title, dy);
```

Comment:
Do we have to declare `getVPadShift` function above and call it here?
It looks like it is not get called from other places.
So perhaps you could remove it to revert the changes here.